sdk/eventhub/Azure.Messaging.EventHubs.CheckpointStore.Blob/src/BlobPartitionManager.cs

```diff
@@ -49,17 +49,19 @@ public sealed class BlobPartitionManager : PartitionManager
         ///   Retrieves a complete ownership list from the storage blob service.
         /// </summary>
         ///
+        /// <param name="eventHubsHostName">The name of the host used to connect to the associated Event Hubs namespace.</param>
```

Comment:
I think we just call this `hostName` within the Event Hub client and other types.   We should probably be consistent here.